app/src/main/java/com/concordium/wallet/ui/base/BaseBindingFragment.kt

```diff
@@ -0,0 +1,66 @@
+package com.concordium.wallet.ui.base
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.activity.OnBackPressedCallback
+import androidx.annotation.CallSuper
+import androidx.annotation.LayoutRes
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.concordium.wallet.util.Log
+
+abstract class BaseBindingFragment<BINDING : ViewDataBinding> : Fragment() {
+
+    protected var _viewDataBinding: BINDING? = null
```

Comment:
`private`?

Reply:
This access should not be restricted as in some cases we need complete control over binding in fragments. The lifecycle bound non-null viewDataBinding is here for convenience and code clarity/quality.